prime-router/metadata/schemas/FL/fl-covid-19.schema

```diff
@@ -251,6 +260,7 @@ elements:
       hl7Field: OBX-17-2
       mapper: "concat(test_kit_name_id, device_id, test_kit_name_id_type)"
       csvFields: [{ name: "Kit^Device^IDType" }]
+      delimiter: ^
 
```

Comment:
Does `delimiter` only apply to csv fields or the concat mapper? If so, I would like it to be put into that. We can talk about it.

Reply:
It applies to the concat mapper. I like having it in Element because we inject Element into the mapper, and I feel like the Element should carry the information about how it wants to be presented/constructed when it's injected.